kf_release_maker/cli.py

```diff
@@ -27,33 +27,32 @@ def cli():
     pass
 
 
-@click.command(name="notes")
-@click.option(
-    "--repo",
-    prompt="The github repository (e.g. kids-first/kf-lib-data-ingest)",
-    help="The github organization/repository to make a release for",
-)
-@click.option(
-    "--release_type",
-    default="minor",
-    type=click.Choice([MAJOR, MINOR, PATCH, PRERELEASE]),
-    prompt="What kind of release is this?",
-    help="Version types follow semantic versioning",
-)
-@click.option(
-    "--project_title",
-    prompt="The title of the project",
-    default="",
-    help="This will be put before the release number in the generated notes",
-)
-@click.option("--github_token", default="", help="API token for GitHub")
-@click.option(
-    "--blurb_file",
-    prompt="Markdown file containing a blurb to prepend to the notes for this release",
-    default="",
-    help="Markdown file containing a blurb to prepend to the notes for this release",
-)
-def release_notes(repo, release_type, project_title, github_token, blurb_file):
+def options(function):
+    function = click.option(
+        "--repo",
+        prompt="The github repository (e.g. kids-first/kf-lib-data-ingest)",
+        help="The github organization/repository to make a release for",
+    )(function)
+    function = click.option(
+        "--release_type",
+        default="minor",
+        type=click.Choice([MAJOR, MINOR, PATCH]),
+        prompt="What kind of release is this?",
+        help="Version types follow semantic versioning",
+    )(function)
+    function = click.option(
+        "--github_token", default="", help="API token for GitHub"
+    )(function)
+    function = click.option(
+        "--blurb_file",
+        prompt="Markdown file containing a blurb to prepend to the notes for this release",
+        default="",
+        help="Markdown file containing a blurb to prepend to the notes for this release",
+    )(function)
+    return function
+
+
+def release_notes(repo, release_type, github_token, blurb_file):
```

Comment:
Shouldn't this be part of `GithubReleaseMaker` and not part of `CLI.py`? Or at the very least, I think it should be a method in the `release_maker.py` module, just not part of `CLI.py`